docs/interpreter/spark.md

```diff
@@ -145,6 +145,11 @@ You can also set other Spark properties which are not listed in the table. For a
     <td>true</td>
     <td>Do not change - developer only setting, not for production use</td>
   </tr>
+  <tr>
+  <td>zeppelin.spark.uiWebUrl</td>
+    <td></td>
+    <td>Overrides Spark UI default URL (ex: http://{hostName}/{uniquePath)</td>
```

Comment:
should this be `{uniquePath)` -> `{uniquePath})`

Reply:
I think we should clarify that the value should be a *full* URL

Reply:
Fixed on latest commit